index/index.html

```diff
@@ -12,9 +12,9 @@
             <tr>
                 <td><input class="searchField" type="text" id="searchField" name="search"></td>
                 <td class="divideLine"></td>
-                <td><button class="searchPrevBtn" type="button" id="searchPrevBtn"></button></td>
-                <td><button class="searchNextBtn" type="button" id="searchNextBtn"></button></td>
-                <td><button class="closeBtn" type="button" id="closeBtn"></button></td>
+                <td><button class="searchPrevBtn" title="Previous" type="button" id="searchPrevBtn"></button></td>
+                <td><button class="searchNextBtn" title="Next" type="button" id="searchNextBtn"></button></td>
+                <td><button class="closeBtn" title="Close find+ bar" type="button" id="closeBtn"></button></td>
```

Comment:
In regular crl+f the title is "Close find bar", not sure if you want to mirror that also?